tests/credentials.json

```diff
@@ -5,7 +5,7 @@
         "credentials": {
             "apiKey": "XXXXXXXX",
             "apiToken": "YYYYYYY",
-            "base_uri": "https://openwhisk.ng.bluemix.net/api/v1",
+            "base_uri": "https://openwhisk.ng.bluemix.net/api/v1"
```

Comment:
please use a more generic uri (i.e. no ibm stuff) 
maybe change to "base_uri": "https://api.service-x.com/v1"